jbmc/unit/solvers/refinement/string_constraint_instantiation/instantiate_not_contains.cpp

```diff
@@ -100,13 +100,13 @@ std::set<exprt> full_index_set(const array_string_exprt &s)
 }
 
 /// Create the cartesian product of two sets.
-template<class T, class U>
-std::set<std::pair<T, U>> product(const std::set<T> ts, const std::set<U> us)
+template <class X, class Y>
```

Comment:
:question: What's the logic here?

Reply:
Template parameter types get renamed to match the name of the parameters to the function, makes sense to me.

Reply:
It should just be made static right - that would fix this? This tool is super neat though!

Reply:
Yeah, it's upset that var `t` and template parameter `T` are only distinguished by case